src/app/home/home.component.html

```diff
@@ -16,103 +16,209 @@
 
 <div class="header">
   <div class="container">
-    <div class="row header1">
-      <div class="col-md-6 splash">
+    <div fxLayout="column wrap" fxLayoutGap="50px" fxLayoutAlign="space-around" class="top-section">
+      <div class="text-center">
         <h1 class="title">Create, Share, Use</h1>
-        <h2 class="title caption">Search Docker Tools and Workflows for the Sciences:</h2>
-        <div class="btn-group col-md-10 col-sm-8 col-xs-12">
-          <input id="searchBar" #search type="text" placeholder="Enter Keyword..." class="search left fully-rounded search-home form-control"
+        <h2 class="title caption">Docker Tools and Workflows for the Sciences</h2>
+        <button *ngIf="!(user$ | async); else loggedIn" mat-flat-button color="accent" routerLink="/login">
+          Create an account
+        </button>
+
+        <ng-template #loggedIn>
+          <button mat-flat-button color="accent" routerLink="/accounts">
+            Welcome {{ (user$ | async).username }}
+          </button>
+        </ng-template>
+
+        <button mat-button routerLink="/search" class="underlined-link">
+          Browse tools and workflows
```

Comment:
Total opinion from me so up to you but it's kind of weird to me to see a button and link ) right next to each other (because they're different formats). Since the search bar is below and "browse tools and workflows" also takes you to the same page, I think I would like it more either

1. Browse tools and workflows wasn't a link anymore, but on a new line and the search bar would be implied that that's how you browse them.
2. Get rid of the browse tools and workflows line and maybe change the text inside the search bar from "Search..." to "Browse tools and workflows"

Reply:
With the `browse tools and workflows` button I was trying to make it less prominent, as the major thing we would want is to get the user to sign up, however I still wanted to make it obvious to the user how to get to the search page. Perhaps a more neutral colour for the `Browse tools and workflows button` and make it a button and not just a link.

The issue with removing the button and only having the search bar is that it requires the user to know what they are searching for. They may not know exactly what they need and are just trying to see what tools/workflows we offer.

Reply:
![browse-button](https://user-images.githubusercontent.com/6331159/57300213-58e71f00-70a4-11e9-9aa5-1fcd390dcff9.png)

Reply:
Ah yeah that makes sense. It definitely looks better to me as a button in that screenshot.

Reply:
I like the button. Will this new user/not logged in view still have the 'My Tools' and 'My Workflows' on the top right?

Reply:
Only logged in users will see my tools/workflows, so a not logged in view will not have the my tools/workflows.